test/reports/test-report-manager.js

```diff
@@ -0,0 +1,290 @@
+const mockgoose = require('mockgoose');
+const mongoose = require('mongoose');
+const assert = require('assert');
+mockgoose(mongoose);
+
+const reportManager = require('../../src/reports/report-manager');
+const reportModels = require('../../src/reports/report-models');
+const Report = reportModels.Report;
+const EventReport = reportModels.EventReport;
+const OtherReport = reportModels.OtherReport;
+
+describe('reportManager', () => {
+  // Mockgoose doesn't start the mock until after 'mongoose.connect()' is
+  // called.
+  // TODO(jmtaber129): Find a version of mockgoose that works and doesn't
```

Comment:
I'll remove this TODO.  This was originally the case, but I found a version that didn't do this and forgot to remove the TODO.